README.md

```diff
@@ -42,14 +42,14 @@ accessing it when you are not sure that it will succeed. The following
 code is unsafe:
 
 ```cpp
-ada::result<ada:url> url = ada::parse<ada:url>("some bad url");
+ada::result<ada:url_aggregator> url = ada::parse<ada:url_aggregator>("some bad url");
```

Comment:
Should we just replace these ada::result's with auto?

Reply:
Yes. One reason to stick with ada::result<ada:url_aggregator> is that some programmers prefer to know exactly what the return type is. But I don't personally care in this instance about how we document it.